R/Functions_L0_auxilary.R

```diff
@@ -2536,46 +2516,6 @@ getDronesCsdGeno <- function(x, nInd = NULL, dronesHaploid = TRUE,
 #'
 #' @return logical
 #'
-#' @examples
-#' founderGenomes <- quickHaplo(nInd = 8, nChr = 1, segSites = 100)
-#' SP <- SimParamBee$new(founderGenomes)
-#' basePop <- createVirginQueens(founderGenomes)
-#'
-#' drones <- createDrones(x = basePop[1], nInd = 1000)
-#' droneGroups <- pullDroneGroupsFromDCA(drones, n = 10, nDrones = nFathersPoisson)
-#'
-#' # Create a Colony and a MultiColony class
-#' colony <- createColony(x = basePop[2])
-#' colony <- cross(colony, drones = droneGroups[[1]])
-#' colony <- buildUp(x = colony, nWorkers = 6, nDrones = 3)
-#' colony <- addVirginQueens(x = colony, nInd = 4)
-#'
-#' apiary <- createMultiColony(basePop[3:4], n = 2)
-#' apiary <- cross(apiary, drones = droneGroups[c(2, 3)])
-#' apiary <- buildUp(x = apiary, nWorkers = 6, nDrones = 3)
-#' apiary <- addVirginQueens(x = apiary, nInd = 5)
-#'
-#' # Caste members taken from Colony class
-#' (tmp <- getCsdGeno(getQueen(colony)))
-#' SIMplyBee:::isGenoHeterozygous(tmp)
-#'
-#' (tmp <- getCsdGeno(getVirginQueens(colony)))
-#' SIMplyBee:::isGenoHeterozygous(tmp)
-#'
-#' (tmp <- getCsdGeno(getWorkers(colony)))
-#' SIMplyBee:::isGenoHeterozygous(tmp)
-#'
-#' # Caste members taken from MultiColony class
-#' (tmp <- getCsdGeno(getQueen(apiary[[1]])))
-#' SIMplyBee:::isGenoHeterozygous(tmp)
-#'
-#' (tmp <- getCsdGeno(getVirginQueens(apiary[[1]])))
-#' SIMplyBee:::isGenoHeterozygous(tmp)
-#'
-#' (tmp <- getCsdGeno(getWorkers(apiary[[1]])))
-#' SIMplyBee:::isGenoHeterozygous(tmp)
-#' # Not exporting this function, since its just a helper and quite specific for
```

Comment:
@janaobsteter This is a defo good comment as we could wonder why we don't export the function